test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Should we add the protocol, or just delete `domain`s/`demoDomain`s that do not have a protocol like `http(s)://`?